api-platform/core/CVE-2025-31481.yaml

```diff
@@ -0,0 +1,14 @@
+title:     "GraphQL query operations security can be bypassed"
+link:      https://github.com/advisories/GHSA-cg3c-245w-728m
+cve:       CVE-2025-31481
+branches:
+    3.4:
+        time:     2025-04-03 15:02:00
+        versions: ['>=3.4.0', '<3.4.17']
+    4.0:
+        time:     2025-04-03 15:02:00
+        versions: ['>=4.0.0', '<4.0.22']
+    4.1:
+        time:     2025-04-03 15:03:00
+        versions: ['>=4.1.0', '<4.1.5']
```

Comment:
@soyuka Can you confirm that this is correct? The advisory does not talk about API Platform 4.1, but from what I see 4.1.5 is the first 4.1 release containing the patch.

Reply:
Correct

Reply:
@soyuka you should probably update the GitHub advisory (both in the repository-level advisory and in their global advisory databases, as they are not automatically synchronized for updates to existing advisories)

Reply:
Oh, and while you are at it, you should probably also adjust the affected 3.4/4.0 versions. Currently it states `<3.4.16` while that should rather be `<=`.